### ISSUE TITLE
Error on world load (crash to main menu) 0.F

### ISSUE THREAD

**Issue by TheEventHorizon** [chaosvolt/cdda-arcana-mod#188]
With the most recent 0.F build (312)
![CDDA Arcana Fatigue error](https://user-images.githubusercontent.com/13070997/124372831-43e58680-dc42-11eb-9214-0a51106b8f1f.PNG)

**Comment by chaosvolt** [comment]
Use the correct version: https://github.com/chaosvolt/cdda-arcana-mod

Read the readme, as it says, copy the Arcana folder into data/mods, make sure you delete your old instance of the Arcana_BN folder if it's still there, then test it. If it's still doing that, then something strange is going on.

**Comment by chaosvolt** [comment]
Okay, I just tested this in 0.F stable. I load-tested it. I debug-learned all spells. Then for good measure I scrolled through the entire spellcasting menu. No errors.

Can't reproduce this.

**Comment by chaosvolt** [comment]
Okay, this is weird. I tested this on 0.F stable and couldn't reproduce this. Then I tested this on today's experimental build and it's there. Searched through and found out the cause, an item spell I'd recently added was set to cast from fatigue and I forgot about it when porting that spell over to the DDA version.

Annoying thing is, the PR that removed that feature was merged months ago, so I don't know why it didn't report this bug until after 0.F stable. Did they only removing the code from it afterwards or something?
